integration/go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/hyperledger/fabric-chaincode-evm/integration
+
+replace github.com/hyperledger/fabric-chaincode-evm/fab3 => ../fab3
+
+replace github.com/hyperledger/fabric-chaincode-evm/evmcc => ../evmcc
+
+require (
+	code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c // indirect
+	github.com/containerd/continuity v0.0.0-20181003075958-be9bd761db19 // indirect
+	github.com/fsouza/go-dockerclient v1.3.0
+	github.com/hyperledger/fabric v1.4.0
+	github.com/hyperledger/fabric-chaincode-evm/fab3 v0.0.0
+	github.com/onsi/ginkgo v1.10.2
+	github.com/onsi/gomega v1.7.0
+	github.com/tedsuo/ifrit v0.0.0-20191009134036-9a97d0632f00
+	github.com/willf/bitset v1.1.10 // indirect
+	go.etcd.io/etcd v3.3.17+incompatible // indirect
+)
+
+replace github.com/go-kit/kit => github.com/go-kit/kit v0.7.0
+
+replace github.com/hyperledger/burrow => github.com/hyperledger/burrow v0.24.4
```

Comment:
I think these two replaces exist due to the v0.x versions might choose non-compiling things otherwise

Reply:
Yup that is part of the need for these replaces. The other is that these are requirements of fab3 so we need to put them as overrides because from go's perspective integration has no dependency because it builds fab3 and doesn't actually use fab3 as a library.